src/data/recipes.js

```diff
@@ -309,3 +309,55 @@ export const popcorn = itemify({
   condition: state =>
     state.itemsSold[items.corn.id] >= 12 && state.itemsSold[butter.id] >= 6,
 })
+
+/**
+ * @property farmhand.module:recipes.bronzeIngot
+ * @type {farmhand.recipe}
+ */
+export const bronzeIngot = itemify({
+  id: 'bronze-ingot',
+  name: 'Bronze Ingot',
+  ingredients: {
+    [items.bronzeOre.id]: 5,
+  },
+  condition: state => state.purchasedSmelter,
+})
+
+/**
+ * @property farmhand.module:recipes.ironIngot
+ * @type {farmhand.recipe}
+ */
+export const ironIngot = itemify({
+  id: 'iron-ingot',
+  name: 'Iron Ingot',
+  ingredients: {
+    [items.ironOre.id]: 5,
+  },
+  condition: state => state.purchasedSmelter,
```

Comment:
Are we sure we want the condition for all of these recipes to be the same (`state.purchasedSmelter`)? It might give the player a little more meaningful work to also have to dig up some number of each ingots' ingredient ores before they can be smelted.

Reply:
i had this same thought too actually so i think that means we should update this to require having dug up a few ores. will update

Reply:
i added a "sold at least 50" to each. since they will each have a different rarity this should make their unlock timings fluctuate nicely

Reply:
50 is an interesting choice here. It's relatively high compared to most other recipes, but I think it's worth exploring how players respond to higher unlock requirements for mining-related items. Let's run with it and give players a challenge! 😄

Reply:
let's also see how it is once we have the ability to actually dig them up. if this ends up being a lot of grinding we should probably reduce it a bit